.github/workflows/master-to-im-master.yml

```diff
@@ -0,0 +1,53 @@
+name: Merge master to im-master PR
+
+on:
+  push:
+    branches:
+      master
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  REPO_NAME: ${{ github.repository }}
+  SOURCE_BRANCH: master
+  TARGET_BRANCH: im-master
+  PR_TITLE: "Automated Pull Request"
+  PR_BODY: "Merge master to im-master"
+  REVIEWERS: '["dshamaev-intermedia", "anikitin-intermedia", "esmitenko-intermedia", "inekrasov-intermedia"]'
+
+jobs:
+  master_to_im_master_pr:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Create Pull Request
+      id: create_pr
+      run: |
+        response=$(curl -v -L \
```

Comment:
Maybe we can extract this scripts into separate sh files and run them here? It will be easier to maintain such scripts

Reply:
OR we can use specific action for that: https://github.com/marketplace/actions/create-pull-request